VultisigApp/VultisigApp/Services/AddressService.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  AddressService.swift
+//  VultisigApp
+//
+//  Created by Enrique Souza Soares on 06/08/24.
+//
+
+import Foundation
+import SwiftUI
+import WalletCore
+
+public class AddressService: ObservableObject {
+    
+    static let shared = AddressService()
```

Comment:
Please remove this singleton implementation , this one doesn't need to be singleton. We should avoid to use singleton whenever we can, not a good pattern